src/Library/demos/List View/main.js

```diff
@@ -0,0 +1,75 @@
+import Gio from "gi://Gio";
+import Gdk from "gi://Gdk";
+import GObject from "gi://GObject";
+import Gtk from "gi://Gtk";
+import Adw from "gi://Adw";
+
+const stack = workbench.builder.get_object("stack");
+const list_view = workbench.builder.get_object("list_view");
+const grid_view = workbench.builder.get_object("grid_view");
+const add = workbench.builder.get_object("add");
+const remove = workbench.builder.get_object("remove");
+const list_box_editable = workbench.builder.get_object("list_box_editable");
+const search_entry = workbench.builder.get_object("search_entry");
+
+//Model
+const string_model = new Gtk.StringList({
+  strings: ["Default Item 1", "Default Item 2", "Default Item 3"],
+});
+
+const selection_model = new Gtk.SingleSelection({ model: string_model });
+
+let item = 1;
+
+const factory_for_list_view = new Gtk.SignalListItemFactory();
+factory_for_list_view.connect("setup", (factory, listItem) => {
+  const listRow = new Adw.ActionRow();
+  listItem.set_child(listRow);
+});
+factory_for_list_view.connect("bind", (factory, listItem) => {
+  const listRow = listItem.get_child();
+  listRow.title = "Demo";
+});
+
+const factory_for_grid_view = new Gtk.SignalListItemFactory();
+factory_for_grid_view.connect("setup", (factory, listItem) => {
+  const listRow = new Adw.ActionRow();
+  listItem.set_child(listRow);
+});
+factory_for_grid_view.connect("bind", (factory, listItem) => {
+  const listRow = listItem.get_child();
```

Comment:
This is one of the confusing bits of these widgets. There are two properties of a `Gtk.ListItem`: `child` (the widget you make) and `item` which is the item from the `Gio.ListModel` (`Gtk.StringList` in our case).

So you want to do something like:

```js
const listRow = listItem.get_child();
const modelItem = listItem.get_item();

listRow.title = modelItem.string;
```

Of course, we'll be using a `Gtk.Box` with a `Gtk.Label` so it would be more like `listRow.child.label = modelItem.string;`, but you get the idea.